crime-applications-adaptor/src/main/resources/schemas/criminalapplicationsdatastore/general/codefendant.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
we are not using this Json as co defendant is out scope for iteration 1, please remove this file

Reply:
same as other comment, see below :)

Reply:
done